packages/caliper-core/test/worker/caliper-worker.js

```diff
@@ -161,5 +161,91 @@ describe('Caliper worker', () => {
             await worker.executeRound(mockTestMessage);
             validateCallsAndWarnings(4);
         });
+
+        it('should complete all transactions', async () => {
```

Comment:
This test isn't really testing the behaviour. Here you are testing the running a fixed number of transactions capability of the worker, so the description needs to match this, it needs to actually prove that it does run the fixed number of transactions and should cover at least 2 different runs ideally. The test should also state that no submit errors will occur so will wait for the transactions to finish.